inst/qml/doeFactorial.qml

```diff
@@ -256,6 +258,40 @@ Form
 				label:							qsTr("General full factorial design")
 				checked:						true
 			}
+
+            RadioButton
+            {
+                id:                             factorialPlackettBurman
+                name:                           "factorialPlackettBurman"
+                label:                          qsTr("Plackett-Burman design")
```

Comment:
this whole part doesn't work yet, right? does it need `debug: true`?

Reply:
This part works, so it doesn't need it